src/DRaaS/SolutionClient.php

```diff
@@ -11,7 +11,7 @@ class SolutionClient extends Client implements ClientEntityInterface
     const MAP = [
         'id' => 'id',
         'name' => 'name',
-        'tenant_uid' => 'tenantUid',
+        'iops_tier_id' => 'iopsTierId',
```

Comment:
Is there a reason that one is being converted to camel case but not the other
```
[tenant_uid] => 4333f8f4-2d47-4856-b5f1-74af0dcab3cd
[iopsTierId] => 75e33c0e-4802-11ea-89fb-005056a64a16
```

Should they all not be snake case to keep the key names consistant?

Reply:
snake case properties are admin only, you're seeing them becasue you tested with an admin APIO  key